### DISCUSSION TITLE
More ways to sort the available driver onboard streams in the race stream selection thing

### DISCUSSION THREAD

**Discussion by Jonoz1** [f1multiviewer/issue-tracker#314] [Feature requests]
A sort button to sort the available driver onboard streams, other than the default alphabetical team's name order, maybe by driver name, by driver number, or by the constructors' standings from the year prior (so Lewis and George 1st and 2nd, Max and Checo 3rd and 4th...)

from
![image](https://user-images.githubusercontent.com/54620172/203856324-a9ac7061-5eed-47ca-bb86-0323686be02b.png)

to?
![very bad ui design](https://user-images.githubusercontent.com/54620172/204002263-a696ce4f-cec9-4c27-914b-b0ec1fdb76e6.png)

I'm sorry I don't know how to design good UI but MV the dev clearly knows how, it's just a general idea of what I'm thinking the sort feature could look like.
I don't like having to scroll down to Red Bull just to get Max's OB, then scroll up for Lando's and so on. I think a feature like this could be such a cool QOL improvement :)

or there was some way to do this and I'm just too stupid to figure it out

**Comment by spambas** [comment]
I Would like to add: to use this sort in right-click menu too

**Comment by Bluebaritone21** [comment]
Scamming people on github, really?

Edit: removed the quoted scam/spam content to prevent spreading. It's been reported to GitHub and the user was banned.
